src/Tribe/Cache.php

```diff
@@ -100,7 +100,9 @@ public function set_transient( $id, $value, $expiration = 0, $expiration_trigger
 	 * @return mixed
 	 */
 	public function get( $id, $expiration_trigger = '', $default = false, $expiration = 0, $args = array() ) {
-		$group = in_array( $id, $this->non_persistent_keys ) ? 'tribe-events-non-persistent' : 'tribe-events';
+		$group = isset( array_flip( $this->non_persistent_keys )[ $id ] ) ?
+			'tribe-events-non-persistent'
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 3cc60c4ce29ea68fef0e72447bf4d184</sub>